bootstrap/_deb_common.sh

```diff
@@ -44,6 +44,10 @@ apt-get install -y --no-install-recommends \
   libffi-dev \
   ca-certificates \
 
+if [[ $? > 0 ]] ; then
```

Comment:
Sorry about the closing branch, I tend to make some silly mistakes from time to time.

And yes it does. I think that should be the correct behaviour though since these packages are supposedly required packages, and failure to install them should exit the installation.